src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -437,12 +454,34 @@
     * @description sees if the contact is NOT the lone member of a household, in which case it will bring up
     * our Merge Household popup.  if they are the lone member, it will just proceed with the HH merge.
     */
-    addOrMergeContact : function(component, conAdd, hhMerge) {
+    addOrMergeContact : function(component, event) {
+        debugger;
```

Comment:
I know this is wip, and does have some debugging code elsewhere, but was this statement intended to be included?  If so, maybe mark it (and other debug code) as `// @debug` so we can find it later and remove it.